Cargo.lock

```diff
@@ -1410,7 +1410,7 @@ dependencies = [
 name = "darwinia-evm-precompile-blake2"
 version = "2.0.0"
 dependencies = [
- "darwinia-evm-primitives",
+ "dp-evm",
```

Comment:
@AurevoirXavier Are you good with this prefix convention?

Reply:
LGTM.

Need a discussion here for further usage. https://github.com/darwinia-network/primitives/issues/1